### ISSUE TITLE
Error when binding in WebJob

### ISSUE THREAD

**Issue by profnimrod** [Azure/azure-signalr#1603]
I have a webjob using Microsoft.Azure.WebJobs 3.0.32.  It contains a function that is bound to a SignalR instance using Microsoft.Azure.WebJobs.Extensions.SignalRService 1.8.0.  This is dependent on Microsoft.Azure.SignalR.Management and Microsoft.Azure.SignalR.  This web job was running perfectly fine when I was using version 1.16.1 of these libraries.  After upgrading to 1.17.0 I started getting the following error for every incoming messages:

Microsoft.Azure.WebJobs.Host.FunctionInvocationException: Exception while executing function: [My function name]
       ---> System.InvalidOperationException: Exception binding parameter 'signalRMessages'
       ---> System.MissingMethodException: Method not found: 'Void Microsoft.Azure.SignalR.ServiceEndpoint.set_Name(System.String)'.
         at Microsoft.Azure.WebJobs.Extensions.SignalRService.OptionsSetup.Configure(ServiceManagerOptions options)
         at Microsoft.Extensions.Options.OptionsFactory`1.Create(String name)

Basically. it no longer works.  Is there a resource that details the breaking changes from 1.16.1 to 1.17.0, or was this unintentional?

The binding expression used within the function signature is:

`[SignalR(HubName = "insyshub")] IAsyncCollector<SignalRMessage> signalRMessages`

**Comment by lbarberic1** [comment]
Did you find any solution to this problem?

**Comment by profnimrod** [comment]
> Did you find any solution to this problem?

Fraid not... surprised to not have had a response from the devs yet though.

**Comment by vicancy** [comment]
caused by breaking change "internal set" removed https://github.com/Azure/azure-signalr/pull/1592/files#diff-e50d03937e2aa54893f89a4c129107f652bd180af87e5a4c32feb7c5c3438424L20

Needs a hotfix.

**Comment by profnimrod** [comment]
Looks like 1.17.1 is now available in Nuget, and it fixes the problem I was seeing.  Many thanks!

**Comment by vicancy** [comment]
Thanks for the confirmation.
